docs/controls/DirectWriteTextBlock.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
We are moving the documentation to https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs. Please open a docs PR there and remove docs from this PR